news/2019-10-20-beatmap-pack-management-update-1.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "Introducing Beatmap Pack Management"
+date: 2019-10-20 19:00:00 +0000
+---
+
+An update of the Beatmap Pack Management project, including new pack releases!
+
+![](/wiki/shared/news/banners/beatmap-packs.jpg)
+
+Our project is called Beatmap Pack Management and we are focused on the development of Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:
+
+I've created a small group of experienced users who are interested in working on handmade charts dedicated to specific themes, ideas, or beatmaps which share similarities. Since July 2019, we've been actively working to provide new theme packs of decent quality with interesting concepts. Interest in this project is very high and some noteworthy members of the community have joined the group to help me by contributing their ideas to this project. We already have a few things to present to you, and I am sure you'll like them.
```

Comment:
`of decent quality` I suggest removing this. 
It makes it sound like "eh this is ok" rather than hyping it up as a cool new project. Focus on what's good about it!

Reply:
simplified that part